cfgov/login/tests/test_auth_forms.py

```diff
@@ -109,105 +98,41 @@ def test_duplicate_emails_allowed_on_user_model(self):
-        self.assertFalse(form.is_valid())
-        self.assertIn(
-            "correct username and password", form.errors["__all__"][0]
-        )
-
-    @override_settings(LOGIN_FAILS_ALLOWED=0)
-    def test_failed_login_lockout(self):
-        """Ensure lockout if our failed logins are over our limit"""
-        form = LoginForm(data={"username": "admin", "password": "badadmin"})
-        self.assertFalse(form.is_valid())
+    def make_login_form_with_expired_password(self):
+        self.user.password_history.all().delete()
 
-        user = User.objects.get(username="admin")
-        self.assertTrue(hasattr(user, "failedloginattempt"))
-
-        with self.assertRaises(ValidationError):
-            form.check_for_lockout(user)
+        with freeze_time("1900-01-01"):
+            PasswordHistoryItem.objects.create(
+                user=self.user, encrypted_password=make_password("mypass")
+            )
 
-    def test_login_locked_out(self):
-        """Ensure we get locked out"""
-        user = User.objects.get(username="admin")
-        TemporaryLockout(
-            user=user, expires_at=(now() + timedelta(hours=1))
-        ).save()
+        return LoginForm(data={"username": "myuser", "password": "mypass"})
 
-        form = LoginForm(data={"username": "admin", "password": "admin"})
+    def test_login_fails_if_user_password_is_too_old(self):
+        form = self.make_login_form_with_expired_password()
         self.assertFalse(form.is_valid())
-        self.assertIn("temporarily locked", form.errors["__all__"][0])
-
-        # Ensure that a temporary lockout doesn't result in a failed login
-        user.refresh_from_db()
-        self.assertFalse(hasattr(user, "failedloginattempt"))
-
-    def test_password_expired(self):
-        """Ensure login is denied if our password is expired"""
-        user = User.objects.get(username="admin")
-        PasswordHistoryItem(
-            user=user,
-            expires_at=now(),
-            locked_until=(now() + timedelta(hours=1)),
-            encrypted_password=make_password("admin"),
-        ).save()
-
-        form = LoginForm(data={"username": "admin", "password": "admin"})
-        self.assertFalse(form.is_valid())
-        self.assertIn("password has expired", form.errors["__all__"][0])
-
-        # Ensure that a password expiration doesn't result in a failed login
-        user.refresh_from_db()
-        self.assertFalse(hasattr(user, "failedloginattempt"))
-
-    def test_clear_failed_login_attempts(self):
-        """Ensure we can clear failed login attempts"""
-        form = LoginForm()
-        user = User.objects.get(username="admin")
-        FailedLoginAttempt(user=user).save()
+        self.assertIn("Your password has expired", form.errors["__all__"][0])
 
-        user.refresh_from_db()
-        self.assertIsNotNone(user.failedloginattempt)
+    def test_superuser_login_succeeds_even_if_password_is_too_old(self):
```

Comment:
This test should be deleted (and the above test simplified) if the custom superuser logic is removed.

Reply:
Done in [df3aac2](https://github.com/cfpb/consumerfinance.gov/pull/7650/commits/df3aac2c721885984228ccfe51fcc7c4804ddb29)